### ISSUE TITLE
adjust-lr

### ISSUE THREAD

**Issue by liuhongxin111** [gy65896/OneRestore#23]
I hope this message finds you well.

In the training command provided in the paper, the learning rate adjustment is set to --adjust-lr 20, while in our current training script we are using --adjust-lr 30.

Could you please advise which value we should use for our training - 20 or 30?

We look forward to your guidance on this matter.

Best regards
